docs/02-get-started/03-deploy-expose-microservice.md

```diff
@@ -206,7 +206,7 @@ EOF
   </summary>
 
```

Comment:
It's outside of the scope of this PR, but it would be good to modify the lines 83-90 here, as:
- setting the ports doesn't exactly save resources, and
- to enter the port numbers, you must first tick the **Expose a separate Service** box